### ISSUE TITLE
NuGet Package from new .csproj - Summary/Title Tag missing

### ISSUE THREAD

**Issue by mishra14** [dotnet/project-system#2937]
_From @mishra14 on October 27, 2017 18:40_

_From @robertmuehsig on October 25, 2017 21:11_

We are using the new .csproj and the integrated NuGet package building feature and I was wondering why I can't specify "Summary" and "Title" within Visual Studio?

When I upload the produced package to nuget.org, a title and summary field is shown without any input and the summary seems kind of important, because it is related to search?!

![image](https://user-images.githubusercontent.com/756703/32023471-8efecb70-b9d9-11e7-868c-4aa52fb10ac1.png)

Repo steps:
* Create new csproj
* fill out some package details
* upload produced package to nuget.org via the Web UI

I guess this is by design, but then the NuGet Gallery should render the fields differently.

_Copied from original issue: NuGet/Home#6102_

_Copied from original issue: Microsoft/VSProjectSystem#258_

**Comment by mishra14** [comment]
_From @robertmuehsig on October 27, 2017 5:34_

Is this part OSS and are PRs possible/welcome?

**Comment by mishra14** [comment]
@robertmuehsig this UI page is not owned by NuGet. I will move your bug to a better repo.

**Comment by mishra14** [comment]
_From @adrianvmsft on November 3, 2017 18:16_

I think you should open this issue on the [dotnet project system github](https://github.com/dotnet/project-system). That is where the implementation of the .net core project system lives.

**Comment by Pilchie** [comment]
The source for the page is at https://github.com/dotnet/project-system/blob/master/src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

**Comment by mishra14** [comment]
@robertmuehsig the page is open source, please take a look at the link in the previous comment.

**Comment by tompazourek** [comment]
MSBuild actually produces warnings that the Summary is missing if you use a long Description. This is what I get from my build output:

```
Issue found with package 'MyTestNuGetPackage1'.
..\..\Program Files\dotnet\sdk\2.0.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets 204 5 MyTestNuGetPackage1

Issue: Consider providing Summary text.
..\..\Program Files\dotnet\sdk\2.0.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets 204 5 MyTestNuGetPackage1

Description: The Description text is long but the Summary text is empty. This means the Description text will be truncated in the 'Manage NuGet Packages' dialog.
..\..\Program Files\dotnet\sdk\2.0.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets 204 5 MyTestNuGetPackage1

Solution: Provide a brief summary of the package in the Summary field.
..\..\Program Files\dotnet\sdk\2.0.2\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets 204 5 MyTestNuGetPackage1
```

**Comment by kkm000** [comment]
Same here, I am getting a long description warning when packaging from an SDK project file. A there is no way to even pass the summary field to the task in `NuGet.Build.Tasks.Pack.targets` which comes with VS15.4.5.
```
C:\Program Files\dotnet\sdk\2.0.3\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(204,5): warning : Description: The Description text is long but the Summary text is empty. This means the Description text will be truncated in the 'Manage NuGet Packages' dialog.
```

So my guess the missing UI field issue belong here, but the missing task parameter is a NuGet issue still. I'll open an issue again there.

**Comment by kkm000** [comment]
Looks like the Summary has been deprecated, so this is a non-issue: https://github.com/NuGet/Home/issues/4587#issuecomment-288913324

@tompazourek: NoPackageAnalysis  to the rescue!
```xml
    <NoPackageAnalysis>true</NoPackageAnalysis>
```

**Comment by davkean** [comment]
This UI wasn't added because the NuGet team asked us not to add it due to both being deprecated. @rrelyea and @mishra14 if that has changed can you please let us know? It looks like `Summary` has been deprecated, but what about `Title`?

**Comment by jjxtra** [comment]
I will just put in my two cents to remove the summary text box all-together then, it is very confusing when publishing nuget why it doesn't carry over...

**Comment by drewnoakes** [comment]
`Summary` is not supported. From [the docs](https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#pack-target):

> `Owners` and `Summary` properties from `.nuspec` are not supported with MSBuild.

`Title` makes sense and will be fixed for the new Project Properties UI in #8065.
